pkg/utils/etcdutil/etcdutil.go

```diff
@@ -840,3 +894,18 @@ func (lw *LoopWatcher) SetLoadTimeout(timeout time.Duration) {
 func (lw *LoopWatcher) SetLoadBatchSize(size int64) {
 	lw.loadBatchSize = size
 }
+
+// CheckWatchChan checks whether the watch channel is blocked for a long time while creating a new watch channel.
+func CheckWatchChan(ctx context.Context, cancel context.CancelFunc, done chan struct{}) {
```

Comment:
What is the difference between checkStream?

Reply:
no difference? only avoid cycle import, how about moving `checkStream` out from `tsoutil`? Which path is suitable?

Reply:
Maybe we can reuse it?

Reply:
move it to grpcutil